Gemfile

```diff
@@ -88,6 +88,7 @@ gem 'sidekiq-instrument'
 gem 'staccato'
 gem 'statsd-instrument'
 gem 'swagger-blocks'
+gem 'tiny_tds', '~> 0.7.0'
```

Comment:
Locking this to such an old version seems like it needs an explanation. The newest is 2.1.2 and there are a lot of changes around TDS client, openssl, etc.

Reply:
Took out tinytds completely. We will be storing the data on the vets api server temporarily while we find out a solution for the sql52 server connection.